.github/workflows/load_test.yml

```
@@ -96,31 +97,58 @@ jobs:
           AZURE_CLIENT_ID: ${{ secrets.CLIENT_ID }}
           AZURE_CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
           AZURE_TENANT_ID: ${{ secrets.TENANT_ID }}
-          CORSO_M356_LOAD_TEST_USER_ID: ${{ secrets.CORSO_M356_LOAD_TEST_USER_ID }}
+          CORSO_M365_LOAD_TEST_USER_ID: ${{ secrets.CORSO_M365_LOAD_TEST_USER_ID }}
         run: |
           go run . exchange emails \
-          --user ${{ env.CORSO_M356_LOAD_TEST_USER_ID }} \
+          --user ${{ env.CORSO_M365_LOAD_TEST_USER_ID }} \
           --destination lt_${{ env.NOW }} \
           --count 10
           go run . exchange contacts \
-          --user ${{ env.CORSO_M356_LOAD_TEST_USER_ID }} \
+          --user ${{ env.CORSO_M365_LOAD_TEST_USER_ID }} \
           --destination lt_${{ env.NOW }} \
           --count 10
           go run . exchange events \
-          --user ${{ env.CORSO_M356_LOAD_TEST_USER_ID }} \
+          --user ${{ env.CORSO_M365_LOAD_TEST_USER_ID }} \
           --destination lt_${{ env.NOW }} \
           --count 10
 
-      # cleanup folders produced by load test
-      - name: Restored Folder Purge
-        if: always()
-        working-directory: ./src
-        env:
-          AZURE_CLIENT_ID: ${{ secrets.CLIENT_ID }}
-          AZURE_CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
-          AZURE_TENANT_ID: ${{ secrets.TENANT_ID }}
-          DELETE_FOLDER_PREFIX: "Corso_Restore_"
-        run: >
-          go run ./cmd/purge/purge.go
-          --user '*'
-          --prefix ${{ env.DELETE_FOLDER_PREFIX }}
+  setup:
+    environment: Testing
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.build.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v3
+      - id: build
+        run: |
+          u=$(echo "${{ secrets.CORSO_M365_LOAD_TEST_ORG_USERS }}" | sed 's/\[/["/g' | sed 's/\]/"]/g' | sed 's/|/","/g')
```

Just a hunch, maybe it is dropping the quotes because of bash. How about we use single quotes for echo? Since the string is expanded by GitHub and not bash, we don't need double quotes anyway.
```suggestion
          u=$(echo '${{ secrets.CORSO_M365_LOAD_TEST_ORG_USERS }}')
```